docker/worker/Dockerfile

```diff
@@ -76,61 +26,33 @@ RUN apt-get update && apt-get -y upgrade && apt-get -y install \
     wget \
     && apt-get clean && rm -rf /var/cache/apt/* /var/lib/apt/lists/*
 
-ADD requirements.txt /tmp/
-RUN cd /tmp/ && pip3 install -r requirements.txt
-
 RUN pip3 install pip --upgrade
-RUN pip3 install requests --upgrade
-RUN pip3 install urllib3 cryptography --upgrade
-
-# Install third-party dependencies
-#   dfwdewey
-#   pyhindsight
-#   impacket
-RUN pip3 install dfDewey
-RUN pip3 install pyhindsight
-RUN pip3 install impacket --no-deps
+RUN pip3 install poetry
 
 # Install various packages from the GIFT PPA
 #   bulkextractor
-#   dfImageTools
 #   docker-explorer
-#   libbde-tools
-#   libfsapfs-tools
-#   libluksde-tools
-#   Plaso
 #   Sleuthkit
 RUN add-apt-repository -y ppa:gift/$PPA_TRACK
 RUN apt-get update && apt-get -y install \
     bulk-extractor \
-    dfimagetools-tools \
     docker-explorer-tools \
-    libbde-tools \
-    libewf \
-    libewf-python3 \
-    libewf-tools \
-    libfsapfs-tools \
-    libluksde-tools \
-    plaso-tools \
-    python3-dfimagetools \
-    python3-dfvfs \
-    python3-plaso \
     sleuthkit \
     && apt-get clean && rm -rf /var/cache/apt/* /var/lib/apt/lists/*
 
 # Add turbinia user to system and sudoers
-RUN useradd -r -s /bin/nologin -G disk,sudo -u 999 turbinia
+RUN useradd -r -s /sbin/nologin -G disk,sudo -u 999 turbinia
 RUN echo "turbinia ALL = (root) NOPASSWD: ALL" > /etc/sudoers.d/turbinia
+RUN echo "Defaults secure_path=\"/venv/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin\"" >> /etc/sudoers.d/turbinia
```

Comment:
Is this the right path?  Does it need to be /home/turbinia/.venv/bin or similar?

Reply:
Do we need /opt/fraken or other dependency paths here?

Reply:
Good catch. Should be ```/home/turbinia/.venv```
I added ```/opt/fraken``` just in case